### DISCUSSION TITLE
svg colors swapped

### DISCUSSION THREAD

**Discussion by sugizo** [niklasf/python-chess#989] [Q&A] [answered]
```
color_0 = {'square light' : 'SeaGreen', 'square dark' : 'DarkSeaGreen'}
chosen_color = color_0
board_svg = chess.svg.board(board = board, colors = chosen_color)
```

**produce in svg**
```
<rect x="0" y="315" width="45" height="45" class="square dark a1" stroke="none" fill="DarkSeaGreen" />
<rect x="45" y="315" width="45" height="45" class="square light b1" stroke="none" fill="SeaGreen" />
```

**when view the svg**
![image](https://user-images.githubusercontent.com/2418295/233144687-1ec8be41-0fc3-4318-9d6d-cb1278ec204a.png)

**question**
how to fix it ?

beside to swapped the color into :
`color_0 = {'square light' : 'DarkSeaGreen', 'square dark' : 'SeaGreen'}`

**Comment by niklasf** [comment, marked as answer]
Weirdly the color SeaGreen is actually defined to be darker than DarkSeaGreen. There's no bug in the rendering.

DarkSeaGreen:

![image](https://user-images.githubusercontent.com/402777/233150244-94aef243-7fe1-4729-b2a9-f14e5e66d988.png)

SeaGreen:

![image](https://user-images.githubusercontent.com/402777/233150184-fa9649a8-7876-42e8-a426-fd08566cc688.png)

**Reply by sugizo** [reply]
yes, you are right, there is no problem in your code, 
because 
like on first post it is seems correct in svg code produced by python-chess

this is the color tested that is work fine with the color name (some are correct, some must be swapped) :
**this must be swapped** 
```
color_1 = {'square light' : 'DarkGray', 'square dark' : 'Gray'} # swapped
color_2 = {'square light' : 'DarkSeaGreen', 'square dark' : 'SeaGreen'} # swapped
```

**this is normal** 
```
color_3 = {'square light' : 'LightGray', 'square dark' : 'Gray'}
color_4 = {'square light' : 'Goldenrod', 'square dark' : 'DarkGoldenrod'}
color_5 = {'square light' : 'Khaki', 'square dark' : 'DarkKhaki'}
color_6 = {'square light' : 'LightSteelBlue', 'square dark' : 'SteelBlue'}
```

**moral of story** 
please test and see first before, 
don't trust the color name

thanks and best regards
